### PR TITLE
Added fixings

### PR DESCRIPTION
**Pull request by og3b**
Ready for a first pass review. Tests aren't passing but good to get some feedback.

### CODE REVIEW

**Comment by og3b** [comment]
@thoozaho not working yet but good to get some feedback.

**Comment by thoozaho** [comment]
Looking good, halfway through. I noticed a bunch of valuation classes in the instrument module have come back? Maybe something to do with the history rewrite?

File: valuation/src/evcel/valuation/RichIndex.scala [hunk 1]
```diff
@@ -127,8 +179,14 @@ case class RichSpotMarketIndex(market : RichSpotMarket) extends RichIndex{
   }
 
   def price(vc : ValuationContext, observationDay : Day) = {
-    // TODO - vc shouldn't need observation day here
-    vc.spotPrice(market.market, observationDay)
+    // TODO AMG - vc shouldn't need observation day here
+    // TODO DC - i think it should as we need it for the fixing?
+    //          Or should vc.fixing take an optional observationDay and
+    //          use vc.marketDay.day by default?
```
Comment (thoozaho) [inline valuation/src/evcel/valuation/RichIndex.scala]: I've forgotten why I thought we shouldn't have observation day. Doesn't seem out of place now.

**Comment by thoozaho** [comment]
@og3b - this looks good.

**Comment by og3b** [comment]
@thoozaho tests pass and I've addressed your comments. I think it's broadly there but there's one TODO in the forward state of the fixing identifier which I'll fix in a later PR.

I think the Index RichIndex split will move around a bit but otherwise feels good. I'd like to merge tonight if you think it's ok?

**Pull request merged**